src/index.html

```diff
@@ -1,13 +1,50 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700& display=swap"
+      rel="stylesheet"
+    >
+  </head>
+
+  <body>
+    <div class="card card--size" data-qa="card">
+      <img
+        class="card__logo"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+      >
+      <section>
+        <h1>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+        <p class="product-code">Product code: 195434</p>
+      </section>
```

Comment:
it's better to use `div` instead of `section`. check other places as well.
The HTML Section element represents a generic section of a document, i.e., a thematic grouping of content, typically with a heading.
```suggestion
      <div>
        <h1>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
        <p class="product-code">Product code: 195434</p>
      </div>
```